HomeWork/src/main/java/exception/ArrayValueCalculator.java

```diff
@@ -0,0 +1,55 @@
+package exception;
+
+import string.StringLevelOne;
+
+import java.util.Arrays;
+import java.util.zip.DataFormatException;
+
+public class ArrayValueCalculator {
+
+    int results = 0;
+    String[][] str = new String[4][4];
+
+    public void arrayInitialization(String[][] strings1) {
+        for (int i = 0; i < strings1.length; i++) {
+            for (int j = 0; j < strings1[i].length; j++) {
+                strings1[i][j] = "5";
+            }
+        }
+    }
+
+    public int doCalc(String[][] strings1) {
+        if (strings1.length != str.length || strings1[0].length != str[0].length) {
```

Comment:
This -> `strings1[0].length != 4` is not correct.
You've started correctly, but haven't finished the validation. You're validating only 1 row here, but it's required to validate them all because the matrix must be of size 4x4 according to the description.
Moreover, you can still reuse `MAX_MATRIX_SIZE`, mentioned above.